distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-intro.adoc

```diff
@@ -165,8 +165,8 @@ It is possible to have only remote Sources configured and no local `CatalogProvi
 
 ===== Product Retrieval
```

Comment:
✏️ I noticed `Product Caching` seems to be different, but I wasn't sure about these (166, 171, 172)

Reply:
I didn't change any of the configurations (metatypes, for example) under the scope of this PR and they use "Product"